frontend/pom.xml

```diff
@@ -39,9 +38,11 @@
             <id>exec-npm-ng-build</id>
             <phase>generate-sources</phase>
             <configuration>
-              <executable>./node_modules/@angular/cli/bin/ng</executable>
+              <executable>npm</executable>
```

Comment:
testowałeś to na jakimś CI? czy na razie tylko lokalnie?

Reply:
lokalnie

Reply:
OK, działa zarówno na Windowsie, jak i na Ubuntu - stworzyłem prostą konfigurację CI, logi i rezultat możesz zobaczyć poniżej